ReelTalk/containers/Lolomo.js

```diff
@@ -24,12 +24,9 @@ var Lolomo = React.createClass({
 
   _showDetails: function(show) {
     this.props.navigator.push({
-      title: show.name,
+      title: 'Show',
```

Comment:
also just saw Greg's comment about lolore so we have already kind of decided to call stuff movies by choosing to use lolomo.... hmmmm something to think about